apps/console/src/features/identity-providers/components/forms/helpers/form-fields-helper.tsx

```diff
@@ -532,7 +646,8 @@ export enum FieldType {
     QUERY_PARAMS = "QueryParameters",
     DROP_DOWN = "DropDown",
     RADIO = "Radio",
-    COPY_INPUT = "CopyInput"
+    COPY_INPUT = "CopyInput",
+    FILEPICKER = "FILEPICKER"
```

Comment:
Shall we follow the same case used to specify other input type here?

```suggestion
    FILEPICKER = "FilePicker"
```

Reply:
Improved with 769cf8391b5176fdb1cbee9cf4cc3fe82c8057d4.